tests/foreman/api/test_http_proxy.py

```diff
@@ -99,8 +99,8 @@ def test_positive_end_to_end(setup_http_proxy, module_target_sat, module_manifes
     assert repo_2.http_proxy_policy == 'global_default_http_proxy'
 
     # Update to selected_http_proxy
-    repo_2.http_proxy_policy = 'none'
-    repo_2.update(['http_proxy_policy'])
+    module_target_sat.api.Repository(id=repo_2.id, http_proxy_policy='none').update()
+    repo_2 = repo_2.read()
```

Comment:
```suggestion
```

Reply:
Will this update the local `repo_2` object without a read? 
Should we assign the update call to `repo_2`?